<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ashwini-kumar" failures="1" tests="5" name="unitTestCase.TC03" time="49.344" errors="0" timestamp="2021-03-21T16:25:06 IST" skipped="0">
  <testcase classname="unitTestCase.TC03" name="EnterLoginCredential" time="17.324"/>
  <system-out/>
  <testcase classname="unitTestCase.TC03" name="LaunchApplication" time="6.907"/>
  <system-out/>
  <testcase classname="unitTestCase.TC03" name="NavigateToLoginPage" time="7.418"/>
  <system-out/>
  <testcase classname="unitTestCase.TC03" name="VerifyHomePageTitle" time="9.130">
    <failure message="expected [QA1 Click Academy | Selenium,Jmeter,SoapUI,Appium,Database testing,QA Training Academy] but found [QA Click Academy | Selenium,Jmeter,SoapUI,Appium,Database testing,QA Training Academy]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [QA1 Click Academy | Selenium,Jmeter,SoapUI,Appium,Database testing,QA Training Academy] but found [QA Click Academy | Selenium,Jmeter,SoapUI,Appium,Database testing,QA Training Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at unitTestCase.TC03.VerifyHomePageTitle(TC03.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- VerifyHomePageTitle -->
  <system-out/>
  <testcase classname="unitTestCase.TC03" name="VerifyLoginPageTitle" time="8.565"/>
  <system-out/>
</testsuite> <!-- unitTestCase.TC03 -->
